src/taxonomy/data/types.mjs

```diff
@@ -31,4 +32,10 @@
  * @typedef {Object} UseQueryResult
```

Comment:
You should not be defining or using `UseQueryResult` here. Please delete this and replace all occurrences of `import("./types.mjs").UseQueryResult` with `import('@tanstack/react-query').UseQueryResult<T>`

Reply:
Or, better yet - see the fixes I made for another PR in https://github.com/open-craft/frontend-app-course-authoring/commit/01c9cd17159f5e4f666d848c02b654c5cd61a3a2 . Basically if you put the right type on the method in `api.js`, then you can leave out the `@returns` definition in `apiHooks.tsx` and the type will still be correct.

https://github.com/open-craft/frontend-app-course-authoring/commit/01c9cd17159f5e4f666d848c02b654c5cd61a3a2#diff-3aff02024d6cb28f96cce104799ab0f3bfe2e9559fe0cd572b9d745daeb590afL25-R25

https://github.com/open-craft/frontend-app-course-authoring/commit/01c9cd17159f5e4f666d848c02b654c5cd61a3a2#diff-b5aca46925ac5f8adac9fbe39002822044019f623908e73f3dba1d5386048dcdL20